mLRS/CommonTx/esp.h

```diff
@@ -227,12 +228,12 @@ void tTxEspWifiBridge::passthrough_do_flashing(void)
     delay_ms(100); // 10 ms is too short, ESP8285 needs more time
     esp_gpio0_high();
     delay_ms(10);
+    com->SetBaudRate(baudrate); // Standard tools should specify 115200 to avoid baudrate change
```

Comment:
the dependency on the if is likely incorrect
I think com baudrate it is always 115200 ... except when it is on the internal module ... so the dependency should be likely the ame a that which selects which comprot is used

why does it have to be 115200? Can't it be the CRSF baudrate?

Reply:
I guess the simplest to do is to just do it always, i.e., to bring it outside of the define

Reply:
> the dependency on the if is likely incorrect I think com baudrate it is always 115200 ... except when it is on the internal module ... so the dependency should be likely the ame a that which selects which comprot is used

Good point.  

> why does it have to be 115200? Can't it be the CRSF baudrate?

The double passthrough makes things a little complicated.  The standard esptool likes to change the backpack baud rate on the fly unless 115200 is specified on the command line  On the other hand, the ELRS version can be told not to change it.  The bottom line is if we want it to work with both the ELRS version of esptool and the combination of ETXinitPassthrough.py as it is today and the standard tools like the Arduino IDE, it's much easier i we use 115200 pretty much everywhere.